create_deb.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+PROGRAM_NAME=sdlhmi
+VERSION=6.1.0
```

Comment:
@LuxoftAKutsan is it related to release 6.1 ? Not 6.2 ?

Reply:
Fixed